.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
-   - "0.10"
-before_install: npm install -g grunt-cli
+   - "4.2"
 install: npm install
```

Comment:
`npm install` is done by default on travis, this line isn't needed. Consider our commit hook repo as a template here: https://github.com/TWExchangeSolutions/jira-precommit-hook/blob/master/.travis.yml

In particular the `sudo: false` which will ensure the builds run faster and are queued for a shorter period of time.